system/jlib/jstats.cpp

```diff
@@ -548,6 +548,21 @@ void validateScope(const char * scopeText)
     }
 }
 
+StatisticScopeType getScopeType(const char * scope)
+{
+    StatsScopeId id;
+    if (nullptr == scope)
+        id.setId(SSTglobal, 0);
+    else
+    {
```

Comment:
needs
```
else if (startsWith(scope, "compile"))
   id.setId(SSTcompilestage, 0))
else
```
because I don't think the function copes with it otherwise.

Reply:
ok will add.